packages/x-components/src/components/currency/base-currency.vue

```diff
@@ -64,8 +64,8 @@
      *
```

Comment:
Can we use this opportunity to update the documentation? I mean, update it to use the new style that we are using in the new components, like in the `my-history.vue`